### PR TITLE
Create portfolio first pull request

### PR DESCRIPTION
**Pull request by Tshobohwa**
# Portofolio Website
In this pull request I have added the html, css, readme, linters files.

### CODE REVIEW

**Review by joskalenda — CHANGES_REQUESTED** [review]
Hi @Tshobohwa , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌

> You have submitted a project, that has not linter GitHub checks actions, they are installed but not running on GitHub action, please fix it and resubmit your work

Your checks




Others checks






Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

-----
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._ 🟢

**Review by Gambit142 — COMMENTED** [review]
Hi @Tshobohwa 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the following issues:
- All linters checks did not pass
- The pull request summary is not descriptive. In your pull request summary, summarize the changes introduced in this project. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+  width: 100%;
+  padding: 5%;
+}
+
+#main-head {
+  width: 80%;
```
Comment (amon-cofie) [inline style.css:44]: - [ ] [OPTIONAL] You have done exceptional work on this css file but I want to leave an advice: Develop and stick to the habit of using class selectors in your css when styling html tags. Ids only select one and only one tag while classes can be used to apply common styles to different elements making class selectors ideal in css. Also as stated in the html and css best practices: 
image

**Review by amon-cofie — APPROVED** [review]
Hi @Tshobohwa 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
🚀 Linters are passing 
🚀 Great use of semantic html
🚀 Design looks very similar to figma template
## Optional suggestions
- Endeavour to always make your PR summaries as detailed as possible, listing everything you did in the branch. You can explore the use of markdown to make your PRs look more professional: checkout [this link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to find out more

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**